migrations/versions/427bdc69b94f_.py

```diff
@@ -0,0 +1,38 @@
+"""add 'description' to suppliers & make 'email' and 'postcode' required fields
```

Comment:
It would be good to have the description of the migrations as part of the filenames - I think it can just be added after the underscore, as with the later scripts in the folder here: https://github.com/alphagov/digitalmarketplace-api/tree/master/migrations/versions